platform/jvm/MacAppNotarizer.kt

```diff
@@ -1,129 +1,51 @@
         return Shell.Command(
-            Shell.Command.arg(XCRUN), Shell.Command.arg(ALTOOL), Shell.Command.arg(NOTARIZE_APP),
-            Shell.Command.arg(PRIMARY_BUNDLE_ID), Shell.Command.arg(options.macAppID),
-            Shell.Command.arg(USERNAME), Shell.Command.arg(options.appleID, printable = false),
-            Shell.Command.arg(PASSWORD), Shell.Command.arg(options.appleIDPassword, printable = false),
-            Shell.Command.arg(FILE), Shell.Command.arg(dmgPath.toString())
-        )
-    }
-
-    private fun parseNotarizeResult(value: String): String {
-        return Regex("RequestUUID = ([a-z0-9\\-]{36})").find(value)?.groupValues?.get(1)
-            ?: throw IllegalStateException("Notarization failed: the response $value from " +
-                    "'xcrun altool --notarize-app' does not contain a valid RequestUUID")
-    }
-
-    private fun waitForPackageApproval(requestUUID: String) {
-        var retries = 0
-        while (retries < MAX_RETRIES) {
-            Thread.sleep(POLL_INTERVAL_MS)
-            val notarizeResult = NotarizationInfoResult.of(
-                shell.execute(notarizationInfoCommand(requestUUID)).outputString()
-            )
-            when (notarizeResult.status) {
-                PENDING -> retries++
-                APPROVED -> {
-                    logger.debug { "$dmgFilename was APPROVED by the Apple notarization service" }
-                    return
-                }
-                REJECTED -> {
-                    throw IllegalStateException("$dmgFilename was REJECTED by the Apple notarization service\n${notarizeResult.rawText}")
-                }
-            }
-        }
-        throw IllegalStateException("Timed out while waiting for $dmgFilename to be scanned by the Apple notarization service; the bundle is still in PENDING status (RequestUUID = $requestUUID)")
-    }
-
-    private fun notarizationInfoCommand(requestUUID: String): Shell.Command {
-        return Shell.Command(
-            Shell.Command.arg(XCRUN), Shell.Command.arg(ALTOOL), Shell.Command.arg(NOTARIZATION_INFO),
-            Shell.Command.arg(requestUUID),
-            Shell.Command.arg(USERNAME), Shell.Command.arg(options.appleID, printable = false),
-            Shell.Command.arg(PASSWORD), Shell.Command.arg(options.appleIDPassword, printable = false)
+                Shell.Command.arg(XCRUN), Shell.Command.arg(NOTARYTOOL), Shell.Command.arg(SUBMIT),
+                Shell.Command.arg(APPLE_ID), Shell.Command.arg(appleCodeSigning.appleID),
+                Shell.Command.arg(PASSWORD), Shell.Command.arg(appleCodeSigning.appleIDPassword, printable = false),
+                Shell.Command.arg(TEAM_ID), Shell.Command.arg(appleCodeSigning.appleTeamID, printable = false),
+                Shell.Command.arg(WAIT), Shell.Command.arg(TIMEOUT), Shell.Command.arg(ONE_HOUR),
+                Shell.Command.arg(dmgPath.toString()),
```

Comment:
Switched from `altool` (deprecated for notarization) to `notarytool`, with different invocation.